ts/model_server.py

```diff
@@ -80,8 +81,9 @@ def start():
                 sys.exit(1)
             ts_conf_file = ts_config
 
-        class_path = \
-            ".:{}".format(os.path.join(ts_home, "ts/frontend/*"))
+        class_path = "{}".format(os.path.join(ts_home, "ts/frontend/*"))
+        if platform.system() != 'Windows':
```

Comment:
platform_path_saparator = {
    "Windows": "",
    "Darwin": ".:",
    "Linux": ".:"
}

class_path = "{}{}".format(platform_path_saparator[platform.system()], os.path.join(ts_home, "ts/frontend/*"))

How about this?

Reply:
Sure, I have modified the code.

Reply:
@harshbafna I have modified the code, please check it.

Reply:
LGTM. thanks.